reflection/src/main/java/com/alevel_hw2/TestJDBCLibrary.java

```diff
@@ -0,0 +1,45 @@
+package com.alevel_hw2;
+
+import java.sql.*;
+
+public class TestJDBCLibrary {
+    public static void main(String[] args) {
+
+        try (Connection connection = DriverManager.getConnection("jdbc:postgresql://localhost:5432/alevel",
+                "postgres", "1234567890")) {
+
+            try (Statement createTableStatement = connection.createStatement()) {
+                String createTableQuery = "CREATE TABLE IF NOT EXISTS employees (" +
+                        "id bigserial PRIMARY KEY," +
+                        "name VARCHAR(255)," +
+                        "position VARCHAR(255))";
```

Comment:
Good, but to apply best practices, consider using PreparedStatement over a simple Statement. In this way, you can reuse your query and pass only values you need in your query.
Move query to some String constant

https://www.javatpoint.com/PreparedStatement-interface